### ISSUE TITLE
🐛 Bug Report: onboarding - first commit doesn't appear on the last commit area and on the commit page

### ISSUE THREAD

**Issue by abrl91** [amplication/amplication#6477]
### What happened?

In the onboarding flow - the first commit doesn't appear on the last commit area and the commit page

### What you expected to happen

I expect to see the commit on the last commit area
![image](https://github.com/amplication/amplication/assets/39680385/13bbf972-4096-4d82-9dac-b8d7b39a540b)

and on the commits page

### How to reproduce

As a new user, create a service, and after the last step, when the code is generated and pushed to GitHub, we should see the commit on the last commit area and the commits page, but it doesn't there. 

### Amplication version

1.7.7

**Comment by KlausMikhaelson** [comment]
Hey @abrl91 is this issue open for community ?

**Comment by abrl91** [comment]
@KlausMikhaelson 
I assigned you. Good luck!

**Comment by KlausMikhaelson** [comment]
Thanks will work on it today!

**Comment by abrl91** [comment]
> @abrl91 what's the situation here ? Is this already solved ?

No. It happens only for new users. When a use creates his first service, the last step is build, and after it finished the commit doesn't appear in the commits page

**Comment by ptdatta** [comment]
@abrl91 If it isn't closed can I work on it?

**Comment by souravjain540** [comment]
@KlausMikhaelson updates?

**Comment by KlausMikhaelson** [comment]
@souravjain540 not any such right now(actually missed it earlier 😅 ) but is it okay if I give an update by thursday ?

**Comment by souravjain540** [comment]
@KlausMikhaelson  👀
